### DISCUSSION TITLE
How to modify the end of this page?

### DISCUSSION THREAD

**Discussion by Shuazijun** [coder/code-server#3207] [Q&A] [answered]
https://imgur.com/SPvkYAN
How to modify the end of this page?I want to remove it.

**Comment by nooobcoder** [comment, marked as answer]
Hey,
        @Shuazijun i do not know **if this would be legal or not** but this is **certainly possible** by modifying a file. I have dont a lot of internal study to do it. I hope this would be patched later. 
Also, @Shuazijun there is nothing wrong if you are using someone's open-source product. You should not be modifying names like this. I think `code-server` is a great product, and just use it originally. Kudos to the developers.
@jsjoeio @code-asher _hats off to you guys being the pillars behind this GOD level project._

To do this open the below file with **sudo rights**
`☁  ~  sudo nano /usr/lib/code-server/lib/vscode/product.json`
![image](https://user-images.githubusercontent.com/50350828/115978747-5cc92000-a59f-11eb-81b6-b72d148aa2e5.png)

Then edit the following two entries,
```
  "nameShort": "coder",
  "nameLong": "coder",
```

![image](https://user-images.githubusercontent.com/50350828/115978760-7c604880-a59f-11eb-8427-2ab7fe778820.png)

**Also, if you feel I have answered your question and your query has been resolved, then please mark the discussion as marked, this would help the moderators to mark discussions that do not required to be discussed farther.**

Thank You,
from **Ankur Paul**  🇮🇳

**Reply by nooobcoder** [reply]
@Shuazijun
